### PR TITLE
chore(tag): fix colors from spec

### PR DESCRIPTION
**Pull request by aledavila**
#### Changelog

**New**

- gray tag color class

**Removed**

- default color

Related (react):
https://github.com/IBM/carbon-components-react/pull/1928

### CODE REVIEW

**Review by laurenmrice — CHANGES_REQUESTED** [review]
The 'Gray' tag needs to be Gray 20 (#DCDCDC)

**Review by laurenmrice — APPROVED** [review]
looks good !! 👍

File: src/components/tag/tag.hbs [hunk 1]
```diff
@@ -8,6 +8,3 @@
 {{#each tags}}
 <span class="{{@root.prefix}}--tag {{@root.prefix}}--tag--{{type}}">{{label}}</span>
 {{/each}}
-{{#if componentsX}}
```
Comment (emyarod) [inline src/components/tag/tag.hbs:11]: just wondering, we're no longer going to have a disabled tag example?

**Review by emyarod — APPROVED** [review]

File: src/components/tag/tag.hbs (hunk 1 above)
Comment (aledavila) [reply src/components/tag/tag.hbs:11]: No. Since its a state it won't be on preview per designer request.

Comment (aledavila) [reply src/components/tag/tag.hbs]: No. It won't be on preview example per designer request

**Review by lovemecomputer — APPROVED** [review]
cool lgtm!

**Pull request merged**